homeassistant/components/proxmox/switch.py

```diff
@@ -0,0 +1,104 @@
+"""Support for switches to turn on/off Proxmox VE VMs/Containers."""
+import homeassistant.components.proxmox as proxmox
+from homeassistant.components.switch import SwitchDevice
+
+DEVICE_CLASS = 'connectivity'
+POWER_ICON = 'mdi:power'
+
+
+async def async_setup_platform(
+        hass, config, async_add_entities, discovery_info=None):
+    """Set up Proxmox VE switch."""
+    nodes = hass.data[proxmox.DATA_PROXMOX_NODES]
+    control = hass.data[proxmox.DATA_PROXMOX_CONTROL]
+    sensors = []
+
+    for name in nodes.keys():
+        item = nodes[name]
+        if 'type' not in item or item['type'] != 'node':
+            if item['control']:
+                sensor = PXMXSwitch(
```

Comment:
Looks weird you call this sensor but adding switches.

Reply:
I renamed it to 'switch'.